PWGJE/EMCALJetTasks/AliAnalysisTaskJetChargeFlavourTemplates.h

```diff
@@ -66,13 +63,13 @@ class AliAnalysisTaskJetChargeFlavourTemplates : public AliAnalysisTaskEmcalJet
   Float_t ParticleJetCharge;      //
   Float_t LeadingTrackPt;         //
   Int_t PdgCode;                  //
+  Float_t ProgenetorFraction;     //

```

Comment:
You've defined `ProgenetorFraction` as a member of your class, so you have to initialize it in your constructor. It has nothing to do with ROOT - this is just standard c++